test/daypickerinput/rendering.js

```diff
@@ -174,5 +174,21 @@ describe('DayPickerInput', () => {
       expect(spy).toHaveBeenCalledTimes(3);
       spy.mockRestore();
     });
+
+    it('should use `selectedDays` from DayPickerInput props', () => {
+      const wrapper = mount(
+        <DayPickerInput
+          selectedDays={new Date(2017, 1, 8)}
```

Comment:
I'm not sure why you are introducing a new `selectedDays` prop here. The idea of this update is that it doesn't matter which day is the user clicking in the overlay: the selected days will be those coming from the `dayPickerProps` if present.